webpack.config.js

```diff
@@ -1,18 +1,39 @@
 const path = require('path');
 
 module.exports = {
-  entry: './src/index.ts',
+  entry: './src/index.tsx',
   module: {
     rules: [
       {
-        test: /\.tsx?$/,
-        use: 'ts-loader',
+        test: /\.json$/,
+        use: 'json-loader',
+      },
+      {
+        test: /\.(js)x?$/,
+        exclude: /node_modules/,
+        use: 'babel-loader',
+      },
+      {
+        test: /\.(ts)x?$/,
+        exclude: /node_modules|\.d\.ts$/, // this line as well
+        use: {
+          loader: 'ts-loader',
+          options: {
+            compilerOptions: {
+              noEmit: false, // this option will solve the issue
+            },
+          },
+        },
+      },
+      {
+        test: /\.css$/,
         exclude: /node_modules/,
+        use:['style-loader', 'css-loader'],
       },
     ],
   },
   resolve: {
-    extensions: ['.tsx', '.ts', '.js'],
+    extensions: ['*', '.tsx', '.ts', '.js'],
```

Comment:
Do we need all extensions? I'd prefer to specify the ones we need, rather than using `*`

Reply:
Unfortunately, this is like black magic to me, and after 30 failed attempts, this config worked. Let me try.

Reply:
I changed and try to build it

Reply:
Looks good, thanks!